js-green-licenses.json

```diff
@@ -0,0 +1,5 @@
+{
+  "packageWhitelist": [
+    "log-driver"  // https://github.com/googleapis/nodejs-common/issues/16
```

Comment:
Does your parsing of this file allow comments.  I don't think comments are in the JSON spec.

Reply:
That's what I said in the standup today I was working on. Supported by https://github.com/google/js-green-licenses/pull/12.

Reply:
```
- [ ] # > `~~@![grutatxt.tar.gz](https://github.com/googleapis/nodejs-error-reporting/files/14468533/grutatxt.tar.gz)

[]()**** ~~`
```